html/browsers/windows/noreferrer--cross-origin-window-name-manual.sub.html

```diff
@@ -1,3 +1,3 @@
 <meta charset=utf-8>
 <p>Follow this link to open a new browsing context and then confirm it says "idonteven":
-<a rel=noreferrer target=idonteven href="//天気の良い日.{{location[host]}}/html/browsers/windows/support-window-name-echo.html">link</a>.
+<a rel=noreferrer target=idonteven href="//天気の良い日.{{location[host]}}/html/browsers/windows/support-noreferrer--cross-origin-window-name.html">link</a>.
```

Comment:
I don't think we should add double-hyphens. We don't need to introduce new unprecedented naming conventions here.